pages/golfers/[id].js

```diff
@@ -0,0 +1,67 @@
+import Layout from '../../components/Layout'
+import ScorePostWidget from '../../components/ScorePostWidget'
+import ScoreCard from '../../components/ScoreCard'
+import { useEffect, useState } from 'react'
+import { getToken, getUsername } from '../../lib/userAuth'
+import { useRouter } from 'next/router'
+import useUserScores from '../../lib/useUserScores'
+import { USER_SCORES_URL } from '../../lib/useUserScores'
+import { useUserName } from '../../lib/useUserName'
+import { USER_URL } from '../../lib/useUserName'
+
+
+
```

Comment:
Personally, I prefer one or two blank lines. It's up to you, but 3 might be too much.